modules/040-control-plane-manager/docs/FAQ_RU.md

```diff
@@ -309,3 +309,187 @@ done
 Для этого вы можете использовать сторонние инструменты резервного копирования файлов, например: [Restic](https://restic.net/), [Borg](https://borgbackup.readthedocs.io/en/stable/), [Duplicity](https://duplicity.gitlab.io/) и т.д.
 
 О возможных вариантах восстановления состояния кластера из снимка etcd вы можете узнать [здесь](https://github.com/deckhouse/deckhouse/blob/main/modules/040-control-plane-manager/docs/internal/ETCD_RECOVERY.md).
+
+## Как из multi-master кластера сделать single-master кластер
+
+1. Выясняем используемую версию и редакцию deckhouse.
+
+```bash
+kubectl -n d8-system get deployment deckhouse \                                                                                                       
+-o jsonpath='Version: {.metadata.annotations.core\.deckhouse\.io\/version}, Edition: {.metadata.annotations.core\.deckhouse\.io\/edition}' \  
+| tr '[:upper:]' '[:lower:]'
+```
+
+2. Запускаем контейнер c dhctl.
+
+```bash
+docker run --pull=always -it -v "$HOME/.ssh/:/tmp/.ssh/" \                                                                                        
+registry.deckhouse.io/deckhouse/<edition>/install:<version> bash
+```
+
+3. Изменяем число реплик для masterNodeGroup.
+
+```bash
+dhctl config edit provider-cluster-configuration --ssh-agent-private-keys=/tmp/.ssh/<SSH_KEY_FILENAME> --ssh-user=<USERNAME> \
+--ssh-host <master-node-name-0>
+```
+
+```yaml
+masterNodeGroup:
+  instanceClass:
+    replicas: 1
+```
+
+4. Снимаем лейблы с нод `<master-node-name-1>`, `<master-node-name-2>`.
+
+```bash
+kubectl label node <master-node-name-1> <master-node-name-2> \
+node-role.kubernetes.io/control-plane- \
+node-role.kubernetes.io/master- \
+node.deckhouse.io/group-
+```
+
+5. Убеждаемся, что ноды пропали из member'ов etcd.
+
+```bash
+kubectl -n kube-system exec -ti $(kubectl -n kube-system get pod -l component=etcd,tier=control-plane -o name | head -n1) -- sh -c \
+"ETCDCTL_API=3 etcdctl --cacert /etc/kubernetes/pki/etcd/ca.crt --cert /etc/kubernetes/pki/etcd/ca.crt --key /etc/kubernetes/pki/etcd/ca.key --endpoints https://127.0.0.1:2379/ member list -w table"
+```
+
+6. Выполняем drain для нод `<master-node-name-1>`, `<master-node-name-2>`.
+
+```bash
+kubectl drain <master-node-name-1> --ignore-daemonsets --delete-emptydir-data
+kubectl drain <master-node-name-2> --ignore-daemonsets --delete-emptydir-data
+```
+
+7. Выключаем (poweroff) `<master-node-name-1>`, `<master-node-name-2>`, удаляем инстансы соотвествующих нод из облака и подключенные к ним диски (kubernetes-data).
+
+8. Удаляем оставшиеся ресурсы с удаленных нод.
+
+```bash
+kubectl delete pods --all-namespaces -o wide --field-selector spec.nodeName=<master-node-name-1>
+kubectl delete pods --all-namespaces -o wide --field-selector spec.nodeName=<master-node-name-2>
```

Comment:
```suggestion
kubectl delete pods --all-namespaces -o name --field-selector spec.nodeName=<master-node-name-1>
kubectl delete pods --all-namespaces -o name --field-selector spec.nodeName=<master-node-name-2>
```

It seems like we should use -o name instead -o wide. I get an error when use `-o wide`: 
- error: unexpected -o output mode: wide. We only support '-o name'

Reply:
How about adding the `--force` option? Or deletion waits too long

Reply:
I added `--force` and remove `-o` param.